engine/src/deposit_event.rs

```diff
@@ -1,13 +1,42 @@
-#[cfg(not(feature = "contract"))]
-use crate::prelude::Vec;
-use crate::prelude::{vec, String, ToString};
-
-use crate::types::*;
-use ethabi::{EventParam, ParamType};
-use primitive_types::U256;
+use crate::prelude::*;
```

Comment:
Yes, I know. But I was inspired by @joshuajbouw

Reply:
But it seems to me that for large code it does not matter. Especially if there are many dependencies. Because without an IDE it is still difficult to determine the dependency. But it just drastically simplifies import, versioning, and management.

Reply:
Sorry, seeing the IDE word, couldn't resist a bikeshched! I have no horse in the race, just want to give some context from my side: 

FWIW, I am personally not a fan on `*` either -- given that today both IntelliJ Rust and rust-analyzer have robust auto-import, the writability benefits of `*` are marginal. Readability wise, explicit imports are better, especially if you are new to the codebase. 

I think the tribal knowledge for `use prelude::*` is that it's only idiomatic when prelude contains trait which extends standard library types. Such are, for example, preludes of [futures](https://docs.rs/futures/0.3.17/futures/prelude/index.html) and [rayon](https://docs.rs/rayon/1.5.1/rayon/prelude/index.html). In contrast, [rand](https://docs.rs/rand/0.8.4/rand/prelude/index.html) prelude is considered unidiomatic, because it contains types and functions in addition to crates.

Reply:
Yeah, I'm in favour of this as we should KNOW whats being imported. Visually, its just easier to read for people so they know where something came from if they are not using an IDE with rust-analyzer or JetBrain's rust plugin.